recipes/optitype/meta.yaml

```diff
@@ -1,31 +1,31 @@
-{% set version = "1.3.2" %}
+{% set version = "1.3.3" %}
 
 package:
   name: optitype
   version: {{ version }}
 
 build:
   noarch: python
-  number: 4
+  number: 0
   script: {{ PYTHON }} -m pip install . --ignore-installed --no-deps -vv
```

Comment:
I'm not sure how this ever worked. Its not a python package with a setup.py file?
Maybe you need to copy the few python scripts into ${PREFIX}/bin ?